ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -878,73 +878,7 @@ def test_tensorflow_is_strictly_increasing(
         fn_tree="math.is_strictly_increasing",
         x=np.asarray(x, dtype=input_dtype),
     )
-
```

Comment:
Why has this been removed in the PR

Reply:
I was trying to figure out where the test of logical_not was failing.